sda-commons-starter/build.gradle

```diff
@@ -0,0 +1,22 @@
+dependencies {
+  compile project(':sda-commons-server-auth')
+  compile project(':sda-commons-server-consumer')
+  compile project(':sda-commons-server-cors')
+  compile project(':sda-commons-server-dropwizard')
+  compile project(':sda-commons-server-jackson')
+  compile project(':sda-commons-server-prometheus')
+  compile project(':sda-commons-server-security')
+  compile project(':sda-commons-server-openapi:')
```

Comment:
No `:` at the end.

Reply:
I have no clue how this came about.. Removed!